python/MooseDocs/extensions/media.py

```diff
@@ -74,65 +52,79 @@ def extend(self, reader, renderer):
         settings['latex_src'] = (None, "Image to utilize when rendering with LaTeX")
+        settings['dark_src'] = (None, "Image to utilize with dark HTML theme")
         settings.update(floats.caption_settings())
         return settings
 
     def createToken(self, parent, info, page):
 
         flt = floats.create_float(parent, self.extension, self.reader, page, self.settings,
                                   bottom=True, **self.attributes)
-        img = Image(flt, src=info['subcommand'], tex=self.settings['latex_src'])
+        img = Image(flt, src=info['subcommand'], dark=self.settings['dark_src'],
+                    tex=self.settings['latex_src'])
         if flt is parent:
             img.attributes.update(**self.attributes)
         return parent
 
 class VideoCommand(command.CommandComponent):
     COMMAND = 'media'
-    SUBCOMMAND = ('ogv', 'webm', 'mp4', 'm4v', 'youtube')
+    SUBCOMMAND = ('ogv', 'webm', 'mp4', 'm4v', None)
 
     @staticmethod
     def defaultSettings():
         settings = command.CommandComponent.defaultSettings()
-        settings['youtube_embed_src'] = (None, "Youtube embed link, only recognized when the subcommand is 'youtube'.")
         settings['latex_src'] = (None, "Image to utilize when rendering with LaTeX")
         settings['controls'] = (True, "Display the video player controls (not compatible with YouTube).")
         settings['loop'] = (False, "Automatically loop the video (not compatible with YouTube).")
         settings['autoplay'] = (False, "Automatically start playing the video (not compatible with YouTube).")
         settings['tstart'] = (None, "Time (sec) to start video.")
         settings['tstop'] = (None, "Time (sec) to stop video.")
+        settings['poster'] = (None, "Add a 'poster' image the the video")
         settings.update(floats.caption_settings())
         return settings
 
     def createToken(self, parent, info, page):
 
         flt = floats.create_float(parent, self.extension, self.reader, page, self.settings,
-                                  bottom=True, img=True)
-
-        if info['subcommand'] == "youtube":
-            vid = Video(flt,
-                        src=self.settings['youtube_embed_src'],
-                        youtube=True,
-                        tex=self.settings['latex_src'],
-                        tstart=self.settings['tstart'],
-                        tstop=self.settings['tstop'])
-        else:
-            vid = Video(flt,
-                        src=info['subcommand'],
-                        youtube=False,
-                        tex=self.settings['latex_src'],
-                        controls=self.settings['controls'],
-                        loop=self.settings['loop'],
-                        autoplay=self.settings['autoplay'],
-                        tstart=self.settings['tstart'],
-                        tstop=self.settings['tstop'])
+                                  bottom=True, img=True, **self.attributes)
```

Comment:
@dschwen Adding the `**self.attributes` was the actual fix to the video size problem